packages/webdriverjs/example-with-callback.ts

```diff
@@ -1,10 +1,11 @@
-const AxeBuilder = require('./lib');
-const WebDriver = require('selenium-webdriver');
+import AxeBuilder from './src';
```

Comment:
IMO we should leave the examples as regular/vanilla JavaScript. This allows them to be copy/pasted and executed without needing a build step.